docs/src/domains/cdn/fastly.md

```diff
@@ -1,58 +1,55 @@
 ---
-title: "Fastly"
+title: "Configure your Fastly CDN"
+sidebarTitle: "Fastly"
 weight: 1
 description: |
-  In some cases you may want to opt to use a CDN such as Fastly rather than the Platform.sh router's cache. Using a CDN can offer a better time-to-first-byte for cached content across a wider geographic region at the cost of the CDN service.
 aliases:
   - "/golive/steps/fastly.html"
 ---
 
-{{% description %}}
+You can [use a CDN](./_index.md) to deliver your site's content to users more quickly.
 
-A Fastly CDN is included for Dedicated projects.
-Platform.sh doesn't offer an integrated CDN on self-service Grid projects at this time,
-but it is a common choice for customers to self-configure.
+By default, a Fastly CDN is included in Dedicated projects.
+Self-Service Grid plans don't automatically include a Fastly CDN, 
+but you can set up one at any time.
 
-Launching a Platform.sh site with Fastly in front of it is nearly the same as launching normally.
-There are only a few notable differences.
-Individual apps may have their own Fastly setup instructions or additional modules.
-Consult the documentation for your app for specific details.
+## Get a DNS TXT record
```

Comment:
This is for Dedicated plans only, but it's not a feature so the tier availability header wouldn't work here. Is there specific formatting you would use in this case to signal that not all clients are concerned?